src/mesh/analysis.rs

```diff
@@ -357,10 +359,11 @@ pub fn are_similar(mesh1: &Mesh, mesh2: &Mesh) -> bool {
 /// normals are identical, because one mesh may reuse (share) vertices and
 /// normals in more faces and the other doesn't (applies to all or some faces).
 ///
-/// They mesh geometries are not necessarily identical in memory but they look
+/// The mesh geometries are not necessarily identical in memory but they look
 /// the same. If the number of vertices differs, the mesh geometries don't share
 /// the same qualities (they are not welded in the same places and at least one
-/// of them is not watertight) are not going to be treated the same by some
+/// of them is not watertight). Despite of that, they are considered to be
```

Comment:
Nit: "Despite that they are considered visually similar, they are not going to be treated the same..."

Reply:
ok